### PR TITLE
Remove inline installation

### PR DESCRIPTION
**Pull request by honeybadgerdontcare**
Remove the [inline installation](https://developer.chrome.com/webstore/inline_installation) of the AMP Validator Chrome Extension from the Validator WebUI.

### CODE REVIEW

File: validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html [hunk 1]
```diff
@@ -74,26 +74,11 @@
...
         &nbsp;&nbsp;&nbsp;
         <a href="https://chrome.google.com/webstore/detail/amp-validator/nmoffdblmcmgeicmolmhobpoocbbmknc"
-           id="install-extension" onclick="return false;" target="_blank">CHROME EXTENSION</a>
+           target="_blank">CHROME EXTENSION</a>
         &nbsp;&nbsp;&nbsp;
         <a href="https://www.npmjs.com/package/amphtml-validator"
            target="_blank">NPM</a>
       </div>
     </paper-toolbar>
   </template>
```
Comment (twifkak) [inline validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html]: Can you delete the imports for polymer and iron-flex-layout above? They don't seem to be referenced in this file, but I dunno if they cause some necessary magic.

**Comment by honeybadgerdontcare** [comment]
@twifkak done, ty.

**Comment by honeybadgerdontcare** [comment]
I thought so too so I looked at our other [files](https://github.com/ampproject/amphtml/blob/master/validator/webui/%40polymer/webui-urlform/webui-urlform.html) and we don't include it. I also tested it and it works. Most likely because these are imported into the main file which has it.

**Comment by twifkak** [comment]
Ah, OK, works for me.

**Pull request merged**